requirements.txt

```diff
@@ -1,3 +1,18 @@
 # Setup and compression
 crunch64>=0.3.1,<1.0.0
 ipl3checksum>=1.2.0,<2.0.0
+
+# asm-differ
+argcomplete
+colorama
+cxxfilt
+python-Levenshtein
+watchdog
+
+# decomp-permuter
+pycparser
+toml
+
+# tools
+mapfile-parser>=2.3.0,<3.0.0
```

Comment:
the ImportError handlers say `mapfile-parser>=1.2.1,<2.0.0`, is mapfile-parser 2.x.x compatible?

Reply:
That is a great question. Unfortunately I don't remember why I wrote this, I'm slowly trying to add code from my old branch. Anecdotally it seems to work with 2.x.x but I put it back to `mapfile-parser>=1.2.1,<2.0.0` for now